docs/nevermined-sdk/compute-example.md

```diff
@@ -0,0 +1,293 @@
+                        container: {
+                            image: 'python',
+                            tag: '3.8-alpine',
+                            checksum:
+                                'sha256:53ad3a03b2fb240b6c494339821e6638cd44c989bcf26ec4d51a6a52f7518c1d'
+                        }
+                    }
+                }
+            }
+
+}
+
+```
+The code to publish the asset is exactly the same. As you can see, to publish different types of assets in Nevermined you just need to set the metadata properly.
+```ts
+const assetAttributes = AssetAttributes.getInstance({
+            metadata: transformMetadata,
+            providers: [config.neverminedNodeAddress]
+})
+algorithmDdo = await nevermined.assets.create(assetAttributes, publisher)
+```
+
+## Publish the Workflow Asset
+
+The last asset we need is the workflow. This asset binds the data with the algorithm(s). A workflow can define multiple stages, each of one defines an input (an array of compute assets), a transformation (an algorithm asset), and an output.
+
+The output will be a new asset published in Nevermined, so as you can see in the output section of the stage we indicate the URLs to publish a new asset in the Nevermined Marketplace API and to access the asset through Nevermined Node.
+
+```ts
+workflowMetadata = {
+            userId,
+            main: {
+                author: 'John Doe',
+                dateCreated: '2019-02-08T08:13:49Z',
+                datePublished: '2019-05-08T08:13:49Z',
+                license: 'CC-BY',
+                name: 'My workflow',
+                type: 'workflow',
+                workflow: {
+                    coordinationType: 'argo',
+                    stages: [
+                        {
+                            index: 0,
+                            stageType: 'Filtering',
+                            input: [
+                                {
+                                    index: 0,
+                                    id: computeDdo.id
+                                }
+                            ],
+                            transformation: {
+                                id: algorithmDdo.id
+                            },
+                            output: {
+                                metadataUrl:
+                                    'http://marketplace.nevermined.localnet/api/v1/metadata/assets/ddo/',
+                                accessProxyUrl: 'http://node.nevermined.localnet',
+                                metadata: {} as any
+                            }
+                        }
+                    ]
+                }
+            }
+}
+```
+
+And the code to publish the asset:
+```ts
+const workflowAttributes = AssetAttributes.getInstance({
+            metadata: workflowMetadata,
+            providers: [config.neverminedNodeAddress]
+        })
+workflowDdo = await nevermined.assets.create(workflowAttributes, publisher)
+```
+
+## Order the Workflow Asset
```

Comment:
Here we are ordering the compute service not the workflow

Reply:
you are right, fixed!